tests/Api/Admin/CatalogPromotionsTest.php

```diff
@@ -43,4 +43,121 @@ public function it_allows_admin_to_get_catalog_promotions(): void
 
         $this->assertResponse($response, 'admin/get_catalog_promotions_admin_response', Response::HTTP_OK);
     }
+
+    /** @test */
+    public function it_allows_admin_to_get_catalog_promotion(): void
+    {
+        $catalogPromotion = $this->loadFixturesFromFiles(['authentication/api_administrator.yaml', 'catalog_promotion.yaml'])['catalog_promotion'];
+
+        $token = $this->logInAdminUser('api@example.com');
+        $authorizationHeader = self::$container->getParameter('sylius.api.authorization_header');
+        $header['HTTP_' . $authorizationHeader] = 'Bearer ' . $token;
+        $header = array_merge($header, self::CONTENT_TYPE_HEADER);
+
+        $this->client->request(
+            'GET',
+            sprintf('/api/v2/admin/catalog-promotions/%s', $catalogPromotion['id']),
```

Comment:
You should use here a code instead of id, as the code is an identifier of catalog promotion entity
```suggestion
            sprintf('/api/v2/admin/catalog-promotions/%s', $fixtures['catalog_promotion']['code']),
```